<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtVigenereCipherMainDialog</class>
 <widget class="QDialog" name="QtVigenereCipherMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VigenereCipher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ToolVigenereCipher.qrc">
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="5">
    <widget class="QLabel" name="label_arrow_up">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolVigenereCipher.qrc">:/images/ToolVigenereCipherArrowUp34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="button_deencrypt">
     <property name="text">
      <string>Deencrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="button_encrypt">
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_arrow_down">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="ToolVigenereCipher.qrc">:/images/ToolVigenereCipherArrowDown34x34.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QLineEdit" name="edit_plaintext">
     <property name="text">
      <string>thisisplaintext</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_key">
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QLineEdit" name="edit_encrypted_text"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="box_key">
     <property name="text">
      <string>abc</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ToolVigenereCipher.qrc"/>
 </resources>
 <connections/>
</ui>
